jekyll/_cci2/(WIP) api-v2-pipelines-parameters-scopes.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: classic-docs
+title: "API v2 Pipelines, Parameters and Scopes"
+short-title: "API v2 Miscellaneous"
+description: "detailed information about API v2 pipelines, parameters, and scopes"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 was designed to utilize pipelines, parameters, and scopes so you can work with API v2 more efficiently.
+
+## Pipeline Parameters
+
+Pipeline parameters are new with API v2. To use pipeline parameters you must use configuration version 2.1 or higher.
+
+### Declaring and using pipeline parameters in configuration
+
+Pipeline parameters are declared using a `parameters` stanza in the top level keys of your `.circleci/config.yml` file. You can then reference the value of the parameter as a config variable in the scope `pipeline.parameters`.
```

Comment:
We'll need to edit this page too I think to include top-level `parameters` https://circleci.com/docs/2.0/configuration-reference/#section=configuration